templates/docker-compose.template

```diff
@@ -5,40 +5,35 @@ services:
   gateway-config:
     image: nebraltd/hm-config:{{CONFIG_VERSION}}
     depends_on:
-      - helium-miner
+      - gateway-rs
       - diagnostics
-      - dbus-session
     environment:
       - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
       - FIRMWARE_SHORT_HASH={{ENV.FIRMWARE_SHORT_HASH}}
-      - DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket
-      - DBUS_SESSION_BUS_ADDRESS=unix:path=/session/dbus/session_bus_socket
     privileged: true
     network_mode: host
     cap_add:
       - NET_ADMIN
     volumes:
       - miner-storage:/var/data
-      - dbus:/session/dbus
     labels:
       io.balena.features.sysfs: 1
       io.balena.features.kernel-modules: 1
-      io.balena.features.dbus: 1
     stop_signal: SIGINT
 
   packet-forwarder:
     image: nebraltd/hm-pktfwd:{{PKTFWD_VERSION}}
     depends_on:
-      - helium-miner
+      - gateway-rs
     restart: always
     privileged: true
     volumes:
       - pktfwdr:/var/pktfwd
     environment:
       - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
 
-  helium-miner:
-    image: nebraltd/hm-gatewayrs:{{GATEWAY_VERSION}}
+  gateway-rs:
```

Comment:
See comment about why this can't be changed

Reply:
It shouldn't be a problem, but I'm not sure. I've removed the dependency, but will make sure packet-forwarder doesn't fail.

Reply:
we will have to keep helium-miner. it will need changes at too many places. Not important right now.

Reply:
Yeah it would need changing in packet forwarder and in hm-pyhelper and not worth the effort.

Also having it called helium-miner is still technically correct